### PR TITLE
Add OC1 and OC2 to default list of acceptors

### LINKED ISSUE #1342: Adding the C-terminus OC1 and OC2 to the default list of hydrogen bond acceptors
In Gromcas, if you use the amber99sb-ildn force field. The two oxygens in the C terminus of the protein are named as OC1 and OC2, which is named as O originally. These two oxygens are part of the carboxylic acid and are hydrogen bond acceptors.
Currently the atom name OC1 and OC2 are not in the default list of hydrogen bond acceptors. I wonder if it is possible to add these two atoms into the default list of hydrogen bond acceptors? As far as I'm concern, the name OC1 and OC2 only represent the C-terminal oxygen.

### Expected behaviour
If the C-terminus of a protein make hydrogen bond with another thing, the hydrogen bond is detected.
### Actual behaviour
It is not.
### Code to reproduce the behaviour

``` python
from MDAnalysis import Universe
from MDAnalysis.analysis.hbonds.hbond_analysis import HydrogenBondAnalysis
from six import StringIO

gro = '''test
3
    1ALA    OC1    1   0.000   0.000   0.000
    2ALA      N    2   0.300   0.000   0.000
    2ALA     H1    3   0.200   0.000   0.000
7.29748 7.66094 9.82962'''
u = Universe(StringIO(gro), format='gro')

h = HydrogenBondAnalysis(u)
h.run()
print(h.timeseries)
# [[]]

h = HydrogenBondAnalysis(u, acceptors = ['OC1',])
h.run()
print(h.timeseries)
# [[[3, 1, 2, 0, 'ALA2:H1', 'ALA1:OC1', 2.0, 180.0]]]

```

### Currently version of MDAnalysis:
py36 mda0.16.1

### PR DESCRIPTION
**Pull request by xiki-tempula**
Fixes #1342  

PR Checklist
------------
 - [x] Tests?
 - [ ] Docs?
 - [x] CHANGELOG updated?
 - [x] Issue raised/referenced?

### CODE REVIEW

**Comment by richardjgowers** [comment]
@xiki-tempula looks good.  can you add a line to the CHANGELOG referencing the issue number too please?

File: package/MDAnalysis/analysis/hbonds/hbond_analysis.py [hunk 1]
```diff
@@ -146,7 +146,7 @@
    =========== ==============  =========== ====================================
    group       donor           acceptor    comments
    =========== ==============  =========== ====================================
-   main chain  N               O
+   main chain  N               O, OC1, OC2
```
Comment (orbeckst) [inline package/MDAnalysis/analysis/hbonds/hbond_analysis.py]: As you said in #1342, OC? is related to Amber99-ildn  in Gromacs. Mention here.

File: package/MDAnalysis/analysis/hbonds/hbond_analysis.py [hunk 2]
```diff
@@ -416,7 +416,7 @@ class HydrogenBondAnalysis(object):
     #: Use the keyword *acceptors* to add a list of additional acceptor names.
     DEFAULT_ACCEPTORS = {
         'CHARMM27': tuple(set([
-            'O', 'OH2', 'OW', 'OD1', 'OD2', 'SG', 'OE1', 'OE1', 'OE2', 'ND1', 'NE2', 'SD', 'OG', 'OG1', 'OH'])),
+            'O', 'OC1', 'OC2', 'OH2', 'OW', 'OD1', 'OD2', 'SG', 'OE1', 'OE1', 'OE2', 'ND1', 'NE2', 'SD', 'OG', 'OG1', 'OH'])),
```
Comment (orbeckst) [inline package/MDAnalysis/analysis/hbonds/hbond_analysis.py]: Technically we should have separate Amber parameters but the way we have been doing this is to lump pretty much everything into CHARMM27. As long as you have a comment in the table above it should be fine.

**Comment by orbeckst** [comment]
Did you do a merge of develop into your branch or did you try a rebase? Either way, there are a bunch of commits in the PR that should not be here: ce213c6,	4e4f90f, aae3100.

**Comment by xiki-tempula** [comment]
@orbeckst Sorry I have rebased to the remote upstream branch. I guess this is a bad idea. I have now rebased based on the commit.

**Comment by orbeckst** [comment]
Sorry, your current history is a bit of a mess. I think the best thing to do is to *create a new branch*, based on the latest develop, and cherry-pick the two commits that you really added in this PR. Something like
```
git checkout develop
git pull
git checkout -b issue-1342-new-acceptors
git cherry-pick 0cff3090489a59d1b4c993f28f18f603a750268b d1d69e5ca0a52060600b136917bf432966e6e988
```
Then create *a new PR based on your issue-1342-new-acceptors branch* (and we will close this one).


(In the future, you would do `git rebase develop; git push -f`.)

**Comment by orbeckst** [comment]
This PR is replaced by #1349

**Pull request closed without merging**